docker-compose.yml

```diff
@@ -11,6 +11,10 @@ services:
     build:
       context: .
       dockerfile: docker/cami-rabbitmq/Dockerfile
+    ports:
+      # Make the RabbitMQ server available on the host so VMs can access it
```

Comment:
I will improve the comments.

The containers are already linked, see `cami-frontend-message-worker` for example. I need to expose the port on the host machine so I can access it with a Vagrant dev machine which I still use as a remote Python interpreter